lib/initalTransactions.ts

```diff
@@ -2,6 +2,8 @@ import fcl from "@onflow/fcl"
 import dotenv from "dotenv"
 import fs from "fs"
 import path from "path"
+
+dotenv.config()
```

Comment:
I'd move this to be right below the `dotenv` import so that it's more clear what it's for.

```ts
import dotenv from "dotenv"
dotenv.config()
```